src/pudl/package_data/eia923/column_maps/byproduct_expenses_and_revenues.csv

```diff
@@ -29,4 +29,4 @@ revenues_fly_bottom_ash_intermingled_1000_dollars,fly_bottom_ash_intermingled_re
 revenues_fgd_byproducts_1000_dollars,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues_thousand_dollars,fgd_byproducts_revenues_thousand_dollars,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues,fgd_byproducts_revenues
 revenues_other_byproducts_1000_dollars,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues_thousand_dollars,other_byproducts_revenues_thousand_dollars,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues,other_byproducts_revenues
 revenues_total_byproduct_1000_dollars,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues_thousand_dollars,total_revenues_thousand_dollars,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues,total_revenues
-early_release,,unnamed_30,,,,,,,,,,,,,,early_release_data_july_2024_not_fully_edited_use_with_caution_do_not_aggregate_to_state_regional_or_national_totals
+early_release,,unnamed_30,,,,,,,,,,,,,,
```

Comment:
What's going on with this weird lil guy from 2009?

Reply:
There's a column in 2009 that has just an "F" next to all records for "final release".